docs/reference/ccr/index.asciidoc

```diff
@@ -195,6 +195,10 @@ You can't manually modify a follower index's mappings or aliases. To make
 changes, you must update the leader index. Because they are read-only, follower
 indices reject writes in all configurations. 
 
+NOTE: Although changes to aliases on the leader index are replicated to follower
+indices, write indices are ignored. Follower indices can't accept direct writes,
+so any indices where `is_write_index` is `true` are not replicated.
```

Comment:
>so any indices where `is_write_index` is `true` are not replicated.

@DaveCTurner, should this be "any indices" or "any index aliases"?

Reply:
The indices are replicated, only the information on the alias about `is_write_index` is not, that flag is always forced to false on the follower.

Reply:
> The indices are replicated, only the information on the alias about `is_write_index` is not, that flag is always forced to false on the follower.

Thanks for the explanation @henningandersen! I updated the note text in 9b5260f943435367546c41852b58cd8349e94d4d -- let me know if that's accurate.